basis/ui/gadgets/theme/theme.factor

```diff
@@ -1,8 +1,97 @@
-! (c)2009, 2010 Slava Pestov, Joe Groff bsd license
+! Copyright (C) 2009, 2010 Slava Pestov, Joe Groff.
+! See http://factorcode.org/license.txt for BSD license.
 USING: io.pathnames sequences ui.images vocabs ;
 IN: ui.gadgets.theme
 
 : theme-image ( name -- image-name )
     "vocab:ui/gadgets/theme/" prepend-path ".tiff" append <image-name> ;
 
+SYMBOL: theme
+SINGLETON: light-theme
```

Comment:
I was afraid of circularity. We need to do `light-theme theme set-global` somewhere and I thought `...themes` was a good place. But `...themes.light` and `...themes.dark` need to import `...themes`. If we define `light-theme` and `dark-theme` in their respective files, `...themes` will need to import `...themes.light` and `...themes.dark` and that's not going to work.